src/main/kotlin/app/extractors/JavaExtractor.kt

```diff
@@ -54,4 +56,28 @@ class JavaExtractor : ExtractorInterface {
 
         return stats
     }
+
+    override fun extractImports(fileContent: List<String>): List<String> {
+        val libraries = mutableSetOf<String>()
+
+        val javaLibraries = File("data/libraries/java_libraries.txt")
```

Comment:
In some Extractors you using list of imports, in some not, why?

Reply:
It depends. In js, there are sometimes no import lines, that's why we tokenize all lines and see what tokens are in the import list. In this case we should catch things like ((com|ru).)?organization.library.modules.* and also skip imports of inner files.

Reply:
So you gonna use this libraries files in a classifier for every language?